event/models.py

```diff
@@ -5,8 +5,8 @@ class Event(models.Model):
 
     title = models.CharField(max_length=255)
     description = models.TextField()
-    start_datetime = models.DateTimeField()
```

Comment:
can we please revert this change? We will keep underscores in python @JavaEria @taleymirza

Reply:
Okay